test/functorch/test_aotdispatch.py

```diff
@@ -956,16 +956,17 @@ def f(a):
             # Thanks to the detach_() AOT Autograd doesn't need to do anything.
             # `out` will show up as having OutputType.non_alias,
             # and ._is_view() == False
-            return out
+            return out, a + 1
```

Comment:
Needed to tweak this test since it's supposed to go down the autograd path, but it happened to only return non-grad tensors